src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -82,12 +82,47 @@ bool ILayoutConstraints.IsHeightConstrained(View requester)
 
 		public double ExtentHeight
 		{
-			get => Content is FrameworkElement fe ? fe.DesiredSize.Height : 0;
+			get
+			{
+				if (Content is FrameworkElement fe)
```

Comment:
Interesting!
Did you add runtime tests addressing this behavior?

Reply:
I won't have time to do it before the merge of this PR.  But sure, I'll add an issue for that.

Reply:
https://github.com/unoplatform/uno/issues/5320